provider/linear/index.ts

```diff
@@ -30,10 +42,21 @@ const linearIssues: Provider<Settings> = {
             return []
         }
 
-        const linearClient = getLinearClient(settingsInput)
-        const issues = await linearClient.issueSearch({ query: params.query, first: 25 })
-
-        return (issues.nodes ?? []).map(issue => ({
+        const query = `
+            query IssueSearch($query: String!, $first: Int!) {
+                issueSearch(query: $query, first: $first) {
+                    nodes {
+                        title
+                        url
+                    }
+                }
+            }
+        `
+        const variables = { query: params.query, first: 25 }
```

Comment:
Do we need to show 25? Probably just 10-15?

Reply:
Yeah, was also thinking about it. I'll update + add an early return for cases when the query is short.